### ISSUE TITLE
Don't let pref setting use isNaN() on the value if it's 'false' or 'true' to decide if it's a Number

### ISSUE THREAD

**Issue by sefeng211** [sitespeedio/browsertime#1254]
https://github.com/sitespeedio/browsertime/blob/master/lib/firefox/webdriver/index.js#L134 is problematic if the value is true or false, because `isNaN` evaluates to false for both cases, so it ends up using `Number` 

We can fix it by doing 
```
if (value === 'false') { value = false; ffOptions.setPreference(name, value); } else if (value === 'true') { value = true; ffOptions.setPreference(name, value); } else if (isNaN(value)) { ffOptions.setPreference(name, value); } else { ffOptions.setPreference(name, Number(value)); }
```

**Comment by soulgalore** [comment]
Hi @sefeng211 great I missed that. I think from the beginning all configurations was just handled as strings as Chrome configs are done, and then I/we added a use case at a time. Let me fix that for the next release.

**Comment by soulgalore** [comment]
Released in 8.7.1, thank you @sefeng211 !
